zuulcilint/checker.py

```diff
@@ -117,7 +115,72 @@ def check_inexistent_nodesets(
                 if nodeset["name"] not in nodeset_list:
                     inexistent_nodesets.add(nodeset["name"])
             except TypeError:
-                if job_nodeset_list.get("name", None) and job_nodeset_list.get("name", None) not in nodeset_list:
+                if (
+                    job_nodeset_list.get("name", None)
+                    and job_nodeset_list.get("name", None) not in nodeset_list
+                ):
```

Comment:
I guess here job_nodeset_list isn't really a list... Maybe add a comment explaining it.

Reply:
renamed to job_nodesets